README.md

```diff
@@ -8,7 +8,5 @@ Inspired by [https://github.com/nijel/rabbitmq-action](https://github.com/nijel/
 
 ```yml
 steps:
-  - uses: baptouuuu/setup-neo4j@v1.0.0
-    with:
-      tag: '3.4'
```

Comment:
The `tag` argument is still available in version 1.2, can you leave it in the doc ? (maybe add the comment `# optional` on the same line)

Reply:
Good point, added it back in